_posts/2017-03-13-P3-Quality.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+asset-type: post
+name: P3 Quality
+title: 'P3 Quality'
+date: 2017-03-13 11:00:00 +00:00
+author: Raquel M Carmena
+image:
+  src: /assets/img/custom/blog/2017-03-13-P3-Quality.jpg
+tags:
+- craftsmanship
+- quality
+
+---
+Do we know what is software quality? Is it measurable? If we want to improve the quality, do we know where to focus on?
+
+Standards ISO/IEC 250xx provide a lot of metrics and guides to measure the quality of a product, but is that enough? What about the process that is followed to create that product? What about the team?
+
+A lot of agile adoptions fail, because they only focus on the process, not on technical excellence.
+
+That’s the reason why we need a global concept of quality, that we’ll call **P3 Quality**. If we want to ensure or improve quality, we should focus on:
+
+* People
+* Product
+* Process
+
+where the most important part is _people_ and others are around us:
```

Comment:
This sentence doesn't make sense. "and others are around us" Are you talking about the people, or about the quality metrics?